pom.xml

```diff
@@ -104,6 +103,18 @@
       <artifactId>maven-resolver-api</artifactId>
       <version>1.6.3</version>
     </dependency>
+    <dependency>
+      <groupId>net.jcip</groupId>
+      <artifactId>jcip-annotations</artifactId>
+      <version>1.0</version>
+      <optional>true</optional>
+    </dependency>
```

Comment:
I don't see any usages of this library elsewhere in this PR. While I don't think this hurts, it is probably simpler to omit it if unused.

```suggestion
```